src/eventProcessor/GoogleAnalyticsEventProcessor.js

```diff
@@ -9,11 +9,11 @@ const logging = goog.require('measurementLibrary.logging');
  * @const {!Object<string,boolean>}
  */
 const TOP_LEVEL_PARAMS = {
-  'client_id': true,
-  'user_id': true,
-  'timestamp_micros': true,
-  'user_properties': true,
-  'non_personalized_ads': true,
+  'clientId': true,
```

Comment:
I assume that the motivation for changing to camel case is because the JSON body uses camel case for these values?

I'd actually rather stick to snake-case for inputs to maintain cross-compatibility with gtag's schema in cases where the keys are the same (e.g. `user_id` and `user_properties` are documented keys in gtag's schema).  Maybe adjusting this to a string->(string|boolean) map for this and automatic params, where a string is the alternate key for transmission.  No need to adjust the config option for additional params though.